File: src/menus/csdlnganh.js
// src/menus/csdlnganh.js
// Logic menu: CSDL Ngành (truong.hcm.edu.vn)
//
// Menu CSDL Ngành có submenu:
// - Ký học bạ

import { openBrowserView } from '../core/browser.js';

export function initCSDLNganh() {
    // Mở hệ thống CSDL Ngành
    const btnCsdl = document.getElementById('btn-csdl');
    if (btnCsdl) {
        btnCsdl.addEventListener('click', function () {
            // Toggle submenu
            if (window.toggleMenu) window.toggleMenu('menu-csdl', this);
            // Mở browser tới trang chính
            openBrowserView(this.dataset.url, this.dataset.name);
        });
    }

    // Submenu items của CSDL
    document.querySelectorAll('#menu-csdl .menu-link[data-nav]').forEach(el => {
        el.addEventListener('click', function () {
            openBrowserView(this.dataset.nav, 'CSDL Ngành');
        });
    });
}

File: src-tauri/src/menus/csdlnganh.rs
// src-tauri/src/scripts/csdlnganh.rs
// JS injection riêng cho hệ thống CSDL Ngành (truong.hcm.edu.vn)
//
// Hệ thống này dùng để:
// - Ký học bạ điện tử
// - Quản lý dữ liệu ngành giáo dục

/// Script bổ sung riêng cho trang CSDL Ngành
/// Ví dụ: hỗ trợ ký học bạ, nhập dữ liệu ngành...
#[allow(dead_code)]
pub fn get_csdl_extra_script() -> String {
    r##"
        (function(){
            // === CSDL NGÀNH - SCRIPT MỞ RỘNG ===
            // Chỗ này sẽ thêm logic xử lý đặc biệt cho CSDL Ngành
            // Ví dụ:
            // - Hỗ trợ ký học bạ hàng loạt
            // - Tự động nhập dữ liệu ngành
            
            console.log('[NSL] CSDL Ngành module loaded');
        })();
    "##
    .to_string()
}
